src/index.html

```diff
@@ -1,2 +1,18 @@
+<HTML>
+	<HEAD>
+		<script type = "text/javascript" src="index.js"></script>
+	</HEAD>
+	<BODY>
```

Comment:
As far as I know, HTML tags are in lowercase, like `<html>` `<body>` and `<head>`.